beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -282,7 +282,7 @@ type CommitteeAssignment struct {
 // than or equal to the minimum valid start slot calculated based on the state's current slot and historical roots.
 func VerifyAssignmentEpoch(epoch primitives.Epoch, state state.BeaconState) error {
 	nextEpoch := time.NextEpoch(state)
-	if epoch > nextEpoch {
+	if epoch > nextEpoch && state.Version() < version.Fulu { // due to fulu not transitioning the state forward and using the proposer look ahead instead
```

Comment:
you shouldn't be able to handle > next epoch even on Fulu

Reply:
That's true but this check assumes that we do a state transition to that epoch which we skip is fulu you can check the other if statement